### ISSUE TITLE
Unable to update webex

### ISSUE THREAD

**Issue by miljanuscumlic** [wimpysworld/deb-get#1102]
## Reporting a bug 🐛

I am using deb-get built from the commit hash 48bc2ab

### Expected behavior

To install the latest version of webex application

### Actual behavior

deb-get displays that there is a new version of the webex package 44.5.0.29672, downloads it, tries to install it, but after I open it, it says its a former version of it 43.11.0.28947. On the next sequence of `deb-get update` and `deb-get upgrade` it tries to install it again. But it never succeeds at it.  

If I remove the package with `deb-get remove webex` and than try to install it with `deb-get install webex` I  again get the version 43.11.0.28947. 

### Steps to reproduce

deb-get update
deb-get upgrade

#### /etc/os-release
```
PRETTY_NAME="Ubuntu 24.04 LTS"
NAME="Ubuntu"
VERSION_ID="24.04"
VERSION="24.04 LTS (Noble Numbat)"
VERSION_CODENAME=noble
ID=ubuntu
ID_LIKE=debian
HOME_URL="https://www.ubuntu.com/"
SUPPORT_URL="https://help.ubuntu.com/"
BUG_REPORT_URL="https://bugs.launchpad.net/ubuntu/"
PRIVACY_POLICY_URL="https://www.ubuntu.com/legal/terms-and-policies/privacy-policy"
UBUNTU_CODENAME=noble
LOGO=ubuntu-logo
```

#### /etc/lsb-release
```
DISTRIB_ID=Ubuntu
DISTRIB_RELEASE=24.04
DISTRIB_CODENAME=noble
DISTRIB_DESCRIPTION="Ubuntu 24.04 LTS"
```

This is an issue on Ubuntu 22.04 also.

#### deb-get version

0.4.0

### deb-get update

```
  [+] Updating /etc/deb-get/01-main
 -=#=-  #      #        #                                                                                                                                                               
  [+] Updating cache of supported apps in the background
Hit:1 http://ubuntu.anexia.at/ubuntu noble InRelease
Hit:2 http://ubuntu.anexia.at/ubuntu noble-updates InRelease
Hit:3 http://ubuntu.anexia.at/ubuntu noble-security InRelease
Hit:4 http://ubuntu.anexia.at/ubuntu noble-backports InRelease
Reading package lists...  [!] ERROR! Missing required information of github package atuin:
                                                                                          URL=
                                                                                              VERSION_PUBLISHED=

  [+] webex (43.11.0.28947) has an update pending. 44.5.0.29672 is available.
 [!] ERROR! Missing required information of github package flameshot:
URL=
VERSION_PUBLISHED=

```
### deb-get upgrade
```
Reading package lists...
Building dependency tree...
Reading state information...
Calculating upgrade...
0 upgraded, 0 newly installed, 0 to remove and 0 not upgraded.
  [+] codium_1.90.0.24158_amd64.deb is up to date.
  [+] gitkraken-amd64.deb is up to date.
  [+] smartgit-23_1_3.deb is up to date.
--2024-06-11 22:58:04--  https://binaries.webex.com/WebexDesktop-Ubuntu-Official-Package/Webex.deb
Resolving binaries.webex.com (binaries.webex.com)... 3.161.119.106, 3.161.119.118, 3.161.119.35, ...
Connecting to binaries.webex.com (binaries.webex.com)|3.161.119.106|:443... connected.
HTTP request sent, awaiting response... 200 OK
Length: 298674334 (285M) [binary/octet-stream]
Saving to: ‘/var/cache/deb-get/Webex.deb’

/var/cache/deb-get/Webex.deb                  100%[=================================================================================================>] 284,84M  2,43MB/s    in 1m 59s  

2024-06-11 23:00:03 (2,40 MB/s) - ‘/var/cache/deb-get/Webex.deb’ saved [298674334/298674334]

  [+] zoom_amd64.deb is up to date.

```

**Comment by philclifford** [comment]
They're now providing the later version in the package. Presuming this is either a transient upstream package/release issue or possibly a stale cache/mirror somewhere..
Can't replicate and looks like the issue was Cisco's not ours in any case.

**Comment by philclifford** [comment]
```
Webex
  Package:	webex
  Repository:	01-main
  Updater:	deb-get
  Installed:	44.5.0.29672
  Published:	44.5.0.29672
```
